source/game/field/obj/objects/Mdush.hh

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "game/field/obj/GeoObject.hh"
+#include <egg/math/Matrix.hh>
+
+namespace Field
+{
+
+class Mdush : public GeoObjectKCL {
+public:
+    Mdush(System::MapdataGeoObj* pMapDataGeoObj) : GeoObjectKCL(pMapDataGeoObj) {}
+    virtual ~Mdush();
```

Comment:
Based off our current convention, how you have it currently kinda implies that we will override this virtual dtor at a later point, but the opposite is true.
```suggestion
    Mdush() override;
```